src/utils/__tests__/snakeToCamel.spec.ts

```diff
@@ -0,0 +1,96 @@
+import { snakeToCamel } from '../snakeToCamel';
+
+describe('snakeToCamel', () => {
+  it('should convert snake_case keys to camelCase', () => {
+    // mock uikit configurations
+    const input = {
+      last_updated_at: 1234567890123,
+      uikit_configurations: {
+        common: {
+          enable_using_default_user_profile: false,
+        },
+        group_channel: {
+          channel: {
+            enable_ogtag: true,
+            enable_typing_indicator: true,
+            enable_reactions: true,
+            enable_mention: false,
+            enable_voice_message: false,
+            reply_type: 'quote_reply',
+            thread_reply_select_type: 'thread',
+            input: {
+              camera: {
+                enable_photo: true,
+                enable_video: true,
+              },
+              gallery: {
+                enable_photo: true,
+                enable_video: true,
+              },
+              enable_document: true,
+            },
+          },
+          channel_list: {
+            enable_typing_indicator: false,
+            enable_message_receipt_status: false,
+          },
+          setting: {
+            enable_message_search: false,
+          },
+        },
+      },
+    };
+
+    const expectedOutput = {
+      lastUpdatedAt: 1234567890123,
+      uikitConfigurations: {
+        common: {
+          enableUsingDefaultUserProfile: false,
+        },
+        groupChannel: {
+          channel: {
+            enableMention: false,
+            enableOgtag: true,
+            enableReactions: true,
+            enableTypingIndicator: true,
+            enableVoiceMessage: false,
+            input: {
+              camera: {
+                enablePhoto: true,
+                enableVideo: true,
+              },
+              enableDocument: true,
+              gallery: {
+                enablePhoto: true,
+                enableVideo: true,
+              },
+            },
+            replyType: 'quote_reply',
+            threadReplySelectType: 'thread',
+          },
+          channelList: {
+            enableMessageReceiptStatus: false,
+            enableTypingIndicator: false,
+          },
+          setting: {
+            enableMessageSearch: false,
+          },
+        },
+      },
+    };
+
+    expect(snakeToCamel(input)).toEqual(expectedOutput);
+  });
+
+  it('should handle empty objects and arrays', () => {
+    expect(snakeToCamel({})).toEqual({});
+    expect(snakeToCamel([])).toEqual([]);
+  });
+
+  it('should preserve non-object and non-array values', () => {
+    expect(snakeToCamel('test')).toBe('test');
```

Comment:
suggestion: good to add test for `camel_case`,

Reply:
Yeah I added it [here](https://github.com/sendbird/sendbird-uikit-react/pull/548/commits/53a68d8b91289f258aa0b88c884c75df5cd0d3a7#diff-f73dfe797e2645768392a250b19223f6cca5439911d3235815b4eb37e7f97cadR84).